plotly/offline/offline.py

```diff
@@ -204,11 +213,30 @@ def _plot_html(figure_or_data, show_link, link_text, validate,
         link_text = link_text.replace('plot.ly', link_domain)
         config['linkText'] = link_text
 
-    script = 'Plotly.newPlot("{id}", {data}, {layout}, {config})'.format(
-        id=plotdivid,
-        data=jdata,
-        layout=jlayout,
-        config=jconfig)
+    if 'frames' in figure_or_data:
+        script = '''
+        Plotly.plot(
+            '{id}',
+            {data},
+            {layout},
+            {config}
+        ).then(function () {add_frames}).then(function(){animate})
```

Comment:
Okay, so after playing `follow-that-code` for a while, I've traced it to the [v2 plot schema](https://api.plot.ly/v2/plot-schema?sha1=22619e9597093bddcc7aad1158615f7607068cd3) Can we add `frames` to this?